spring-integration-jmx/src/test/java/org/springframework/integration/jmx/config/MBeanExporterParserTests.java

```diff
@@ -125,13 +127,13 @@ private void checkCustomized(MessageChannelMetrics metrics) {
 				.isCloseTo(.001 / 140000, Offset.offset(.01));
 	}
 
-	private void checkCustomized(MessageHandlerMetrics metrics) {
+	private void checkCustomized(org.springframework.integration.support.management.MessageHandlerMetrics metrics) {
 		assertThat(TestUtils.getPropertyValue(metrics, "handlerMetrics.duration.window")).isEqualTo(20);
 		assertThat(TestUtils.getPropertyValue(metrics, "handlerMetrics.duration.factor", Double.class))
 				.isCloseTo(1000000., Offset.offset(.01));
 	}
 
-	public static class CustomMetrics implements MetricsFactory {
+	public static class CustomMetrics implements org.springframework.integration.support.management.MetricsFactory {
```

Comment:
Can we consider do not do this in the tests any more?
Maybe it ever is going to be a fact to remove this test altogether since it covers a deprecated functionality...